build.gradle

```diff
@@ -13,11 +13,12 @@ buildscript {
 plugins {
     alias(libs.plugins.detekt)
     alias(libs.plugins.ben.manes)
+    alias(libs.plugins.ksp) apply false
```

Comment:
Why is apply false here? Is the point to just add the plugin to the namespace so it can be applied at the module level?

Reply:
If I didn't apply false this plugin would be applied to our root project which isn't what it's meant for _but_ since I applied it here with a version number I can just apply the ID at the module level and maintain the version in our libs file.  If I just applied it at each individual module level I would have to do `apply plugin: 'com.google.devtools.ksp' version "1.5.31-1.0.1"` for each module and the would be annoying to maintain.